### PR TITLE
Refactor backend Zeus setup

### PR DESCRIPTION
**Pull request by META-DREAMER**
Simplified backend setup with better code organize and closer alignment with updated client side implementation. Queries and mutations are now explicitly separated. Backend generated code is now in `api-lib` instead of `src/lib`

### CODE REVIEW

File: scripts/db-seed.ts [hunk 1]
```diff
@@ -368,8 +380,8 @@ function fakeCircleGifts(
     }
     const getEpoch = createSampler(availableEpochs, epochSkew);
 
-    const getSender = createSampler(users, giveSkew);
-    const getRecipient = createSampler(users, recieveSkew);
+    const getSender = createSampler(users || [], giveSkew);
+    const getRecipient = createSampler(users || [], recieveSkew);
```
Comment (CryptoGraffe) [inline scripts/db-seed.ts:384]: just noticed typo here in `recieveSkew` but its not new here

**Review by CryptoGraffe — APPROVED** [review]
lg2m just one comment that isn't new

**Review by zashton — APPROVED** [review]
thanks for the zeus cleanup

**Review by topocount — APPROVED** [review]
nice, clean separation of concerns

**Pull request merged**